src/pluto/grayscale_image.cr

```diff
@@ -54,10 +54,22 @@ class Pluto::GrayscaleImage < Pluto::Image
   end
 
   def to_rgba
-    RGBAImage.new(@gray.clone, @gray.clone, @gray.clone, Array(UInt8).new(size) { 1u8 }, width, height)
+    RGBAImage.new(@gray.clone, @gray.clone, @gray.clone, Array(UInt8).new(size) { 255u8 }, width, height)
   end
 
   def size : Int32
     @width * @height
   end
+
+  def mask_from(&block : (Int32, Int32, UInt8) -> Bool) : Mask
+    Mask.new(width, BitArray.new(size) do |i|
+      block.call(i % width, i // width, @gray[i])
+    end)
+  end
```

Comment:
Tricky part - creating a `Mask` from a grayscale image is easy because the grayscale image only has a single channel. I've opted to not try and create a `mask_from` method from the `RGBAImage` because I wasn't sure what the expected behavior would be for which channel to use. We could return a mask for _each_ channel, or let a `ChannelType` be specified, but.... then I decided to only support grayscale images.